aiml40/absa/notebooks/absa.ipynb

```diff
@@ -52,7 +52,7 @@
     "#   \n",
     "# Step 1 - Setup your environment\n",
     "#   \n",
-    "![title](images/step1.png)"
+    "![title](https://raw.githubusercontent.com/microsoft/aiml40/master/aiml40/absa/notebooks/images/step1.png)"
```

Comment:
these links don exist - the repo is: microsoft/ignite-learning-paths-training-aiml/aiml40 ...

Reply:
links now updated